backend/amd64/emit.mlp

```diff
@@ -718,18 +718,24 @@ let emit_global_label s =
 let move (src : Reg.t) (dst : Reg.t) =
   if src.loc <> dst.loc then
     begin match src.typ, src.loc, dst.typ, dst.loc with
-    | Float, Reg.Reg _, Float, Reg.Reg _
-    | Vec128, Reg.Reg _, Vec128, Reg.Reg _ -> I.movapd (reg src) (reg dst)
+    | (Float | Vec128), Reg.Reg _, (Float | Vec128), Reg.Reg _ ->
```

Comment:
This should stay as before, only `float,float` and `vec128,vec128` pairs are allowed.